### PR TITLE
Update Generator.php

### PR DESCRIPTION
**Pull request by WinterSilence**
Update preserved keywords

| Q             | A
| ------------- | ---
| Is bugfix?    | yes
| New feature?  | no
| Breaks BC?    | no
| Tests pass?   | yes

### CODE REVIEW

**Review by samdark — APPROVED** [review]
Looks good. Please add a line for CHANGELOG.md. Thanks.

**Comment by WinterSilence** [comment]
@samdark #445

**Comment by samdark** [comment]
Please push to the same branch as in this PR. No need for separate PR.

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Yii Framework 2 gii extension Change Log
 2.2.2 under development
 -----------------------
 
-- no changes in this release.
+- Chg #444: update preserved keywords in generator (WinterSilence)
```
Comment (bizley) [inline CHANGELOG.md:7]:
```suggestion
- Enh #444: Updated reserved keywords in generator (WinterSilence)
```

**Review by bizley — CHANGES_REQUESTED** [review]

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
👍

**Pull request merged**